pallets/creditcoin/src/types/loan_terms.rs

```diff
@@ -122,16 +121,21 @@ where
 }
 
 #[cfg(test)]
-mod tests {
-	use super::calc_interest;
-	use crate::ExternalAmount;
-	use ethereum_types::U256;
-
-	#[test]
-	pub fn test_calc_interest() {
-		let principal_amount = ExternalAmount::from(100_000u64);
-		let interest_rate_bps = 1_000;
-		let interest = calc_interest(&principal_amount, interest_rate_bps);
-		assert_eq!(interest, U256::from(10_000u64));
+impl<Moment> Default for LoanTerms<Moment>
+where
+	Moment: sp_runtime::traits::One,
+{
+	fn default() -> Self {
+		Self {
+			amount: Default::default(),
+			interest_rate: InterestRate { rate_per_period: 0, decimals: 1, period_ms: 1 },
+			term_length_ms: Moment::one(),
```

Comment:
Isn't it weird that the default LoanTerms does not have a default InterestRate?

Reply:
@CAGS295 are you saying `InterestRaate` should have its own default implementation?